modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -57,17 +57,22 @@ For information on the **<Open source plugin name>** plugin see xref:<plugincode
 
 The following premium plugin updates were released alongside {productname} <x.y[.z]>.
 
-=== <Premium plugin name 1> <Premium plugin name 1 version>
+=== Mentions
 
-The {productname} <x.y[.z]> release includes an accompanying release of the **<Premium plugin name 1>** premium plugin.
+The {productname} 7.0 release includes an accompanying release of the **Mentions** premium plugin.
 
-**<Premium plugin name 1>** <Premium plugin name 1 version> includes the following <fixes, changes, improvements>.
+**Mentions** Premium plugin includes the following improvement.
 
-==== <Premium plugin name 1 change 1>
+==== Autocompleter did not work when attempting to insert a `mention` or `emoji` when using an `IME`.
+//# TINY-10637
 
-// CCFR here.
+Previously when attempting to insert a **mention** or **emoji** using an `IME` on Android and using Korean, users encountered an obstacle where it wasn't possible to complete the insertion.
+
+{productname} 7.0 addresses this issue, by rewriting the handler for `autocompleter`, ensuring that it no longer interferes with the DOM during composition with an IME.
+
+As a result, users can now insert **mentions** and **emojis** using the `autocompleter`.
```

Comment:
This sounds like an autocompleter change, rather than a mentions change.

Reply:
Moved to "Bug fixes" section